docs/getting-started/installation.md

```diff
@@ -43,6 +43,7 @@ For various reasons, certain hosting providers may have other steps that should
 - [Digital Ocean Installation Notes](/dokku/getting-started/install/digitalocean)
 - [Linode Installation Notes](/dokku/getting-started/install/linode/)
 - [Microsoft Azure Installation Notes](/dokku/getting-started/install/azure/)
+- [DreamHost Cloud Installation Notes](/dokku/getting-started/install/dreamhost/)
```

Comment:
Can you move this up above the linode documentation so it is in alphabetical order?

Also, please add a link on [this line](https://github.com/dokku/dokku/blob/master/docs/home.html#L173) as well. It'll appear on our homepage.